.gitattributes

```diff
@@ -0,0 +1 @@
+**/src/test/resources/**/*.java.generated linguist-generated=true
```

Comment:
https://docs.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github

This collapses generated code in diffs by default.